Issue #42: Fixed errors after conflict resolvings, switched back to the only settings module, added reading of dev- and security-related parameters from the environment, integrated changes from the issue #33.

lilliputten opened the pull request:
- Auto prettified code to meet 'blue' prettifier standard. Removed extra unused code. Updated configuration.
- Fixed previous errors.
- Using only settings module (`dds_registration/settings.py`). Fixing DEV/LOCAL/DEBUG flags, deriving these vars from the environment (or `.env` file). Added npm script command to run dev server (`start-django-dev`). Added original site url (`events.d-d-s.ch`) to default `ALLOWED_HOSTS` list.
- Integrated changes from issue #33 (branch `33-templates-folder`) -- they're included some changes for app settings.
- Updated import paths to local version of `core` after wrong conflicts resolving.
- Updated environment vars reading (added explicit reading of `.env` file), extended check for expected required variables from the environment, removed duplicated apps from `INSTALLED_APPS` (probably these were the effects of conflicts resolves).

cmutel committed:
Split dev and prod settings

cmutel committed:
Linting

cmutel committed:
Fixes for production settings

lilliputten committed:
Issue #33: Moved source vendor files to `src` folder.

lilliputten committed:
Issue #33: Assets files moved to `src` folder.

lilliputten committed:
Issue #33: Update paths (to remove template includes from static folder). Source files moved to the folder `src`.

lilliputten committed:
Issue #33: Minor changes.

lilliputten committed:
Issue #33: Updated version: 0.0.11.

lilliputten committed:
Issue #42: Auto prettified code to 'blue' prettifier standard. Removed extra unused code. Updated configuration.

lilliputten committed:
Issue #42: Fixed previous errors.

lilliputten committed:
Issue #42: Using only settings module (`dds_registration/settings.py`). Fixing DEV/LOCAL/DEBUG flags, deriving these vars from the environment (or `.env` file). Added npm script command to run dev server (`start-django-dev`). Added original site url (`events.d-d-s.ch`) to default `ALLOWED_HOSTS` list.

lilliputten committed:
Issue #42: Integrated changes from issue #33 (branch `33-templates-folder`) -- they're included some changes for app settings.

lilliputten committed:
Issue #42: Updated import paths to local version of `core` after wrong conflicts resolving.

lilliputten committed:
Issue #42: Updated environment vars reading (added explicit reading of `.env` file), extended check for expected required variables from the environment, removed duplicated apps from `INSTALLED_APPS` (probably these were the effects of conflicts resolves).

cmutel commented:
> Auto prettified code to meet 'blue' prettifier standard. Removed extra unused code. Updated configuration.

Any chance you can switch over to black? Otherwise we will keep overwriting each other's code :)

cmutel commented:
The single settings file is nice.

lilliputten commented:
> > Auto prettified code to meet 'blue' prettifier standard. Removed extra unused code. Updated configuration.
> 
> Any chance you can switch over to black? Otherwise we will keep overwriting each other's code :)

I tried to ask about the reasons for these massive changes (mostly for quotes only) but probably lost the answer or it wasn't answered.

I used blue only because of the ability to have settings for single quotes (it made much less visual garbage), but if it (to have double quotes) is crucial, I can try to cope with it.
